inc/saiswitch.h

```diff
@@ -1269,6 +1269,30 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_ACL_STAGE_EGRESS,
 
     /**
+     * @brief Max number of Segments in SID List supported
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_SEGMENTROUTE_MAX_SEGMENTS,
+
+    /**
+     * @brief Max number of SID Lists supported
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_SEGMENTROUTE_MAX_SIDLISTS,
+
+    /**
+     * @brief List of Type Length Value types supported for source
+     *
+     * @type sai_s32_list_t sai_tlv_type_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_SEGMENTROUTE_TLV_TYPE,
+
```

Comment:
Its not maximum, its the actual individual TLV types a vendor can support.  For example if a vendor only supports Ingress Node TLV type but not all others, it would pass Ingress Node TLV enum in